### ISSUE TITLE
Take calibration shots with other iCubie

### ISSUE THREAD

**Issue by devangel77b** [usna-ew309/corona#13]

**Comment by devangel77b** [comment]
Hi Everyone:

I processed Dennis's calibration and things look pretty nifty! 

Dennis, did you calibrate three separate iCubies or the same iCubie three times? 

The calibration intrinsics are below; and I've summarized the calibrations in the attached files.

Calibration Data 1:
A_c2m =
[ 670.201454,   0.000000, 330.540641 ]
[   0.000000, 667.814301, 231.190287 ]
[   0.000000,   0.000000,   1.000000 ]
Calibration Data 2:
A_c2m =
[ 668.387417,   0.000000, 332.154522 ]
[   0.000000, 667.774294, 232.872920 ]
[   0.000000,   0.000000,   1.000000 ]
Calibration Data 3:
A_c2m =
[ 673.466007,   0.000000, 332.817549 ]
[   0.000000, 671.651322, 228.415883 ]
[   0.000000,   0.000000,   1.000000 ]

- Mike

**Comment by devangel77b** [comment]
To get info on variation among cameras, I collected 30 calibration shots each using Kutzer;s calibration object, 0.75-inch 19.05mm 6 wide by 7 tall checkerboard object.  Each set of 30 was taken using a different iCubie camera randomly selected from the box in Ri080. 
https://drive.google.com/drive/folders/1H55iMqm_VxcQ_hRONkG-qcJdPwDgZUiY 

I expect the focal length and sensor pitch to be pretty much the same for all of them, with some variation in the principal point. 

The 5 cameras are different from the one I used for the first set of shots (that camera is the one normally at the instructor station in Ri078).
